code/js/controllers/SpotifyController.js

```diff
@@ -4,32 +4,20 @@
 
   var controller = new BaseController({
     siteName: "Spotify",
-    play: ".now-playing-bar button[class*=play]",
-    pause: ".now-playing-bar button[class*=pause]",
-    playNext: ".now-playing-bar button[class*=skip-forward]",
-    playPrev: ".now-playing-bar button[class*=skip-back]",
-    like: ".now-playing-bar button[class*=heart]",
+    play: ".now-playing-bar button[title=Play]",
```

Comment:
Although these selectors look neat and readable, I think using the `nth-child` pseudoclass would be a better solution.

When you have a browser with non-English locale as a main locale, Spotify uses this locale to display UI elements; and all element titles are locale-dependent. That means these selectors won't work in cases when a user selected a non-English browser locale. For example, I've a Russian locale selected in my browser, and here's an outer HTML of the play button:

```html
<button
  class="_82ba3fb528bb730b297a91f46acd37a3-scss"
  title="Слушать"
  aria-label="Слушать"
>
  ...
</button>

```

I think using the `nth-child` pseudoclass is a good alternative, though it's not well readable, as the selectors added in this PR. The button order remains the same, so it should be no problem with the `nth-child` parameters.

What are your thoughts on this?

Reply:
I totally forgot about localization 🤦. I'll make a commit switching to nth-child later today (unless you want to).

Reply:
Since the play and pause buttons are now the same (same selector), does anything else need to change?

Reply:
There's also the possibility to use the localized aria-labels, see this PR for chrome-media-keys: https://github.com/msfeldstein/chrome-media-keys/pull/140